public/index.html

```diff
@@ -5,7 +5,25 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <meta name="referrer" content="same-origin">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="description" content="qBittorrent Web UI" />
+    
+    <!-- Chrome, Opera, and Firefox OS -->
+    <meta name="theme-color" content="#4D8AD5" />
+    <!-- Windows Phone -->
+    <meta name="msapplication-navbutton-color" content="#4D8AD5" />
+    
+    <link rel="icon" type="image/x-icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="apple-touch-icon" href="/apple-touch-icon.png" />
+    <link rel="apple-touch-icon" sizes="57x57" href="<%= BASE_URL %>/apple-touch-icon-57x57.png" />
+    <link rel="apple-touch-icon" sizes="72x72" href="<%= BASE_URL %>/apple-touch-icon-72x72.png" />
+    <link rel="apple-touch-icon" sizes="76x76" href="<%= BASE_URL %>/apple-touch-icon-76x76.png" />
+    <link rel="apple-touch-icon" sizes="114x114" href="<%= BASE_URL %>/apple-touch-icon-114x114.png" />
+    <link rel="apple-touch-icon" sizes="120x120" href="<%= BASE_URL %>/apple-touch-icon-120x120.png" />
+    <link rel="apple-touch-icon" sizes="144x144" href="<%= BASE_URL %>/apple-touch-icon-144x144.png" />
+    <link rel="apple-touch-icon" sizes="152x152" href="<%= BASE_URL %>/apple-touch-icon-152x152.png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="<%= BASE_URL %>/apple-touch-icon-180x180.png" />
```

Comment:
BASE_URL can be removed. https://www.w3schools.com/Html/html_filepaths.asp for more info.

Reply:
@bbogdanov No, it shouldn't be removed. Otherwise it won't work when deploy to non-root path and some other situation.

Reply:
Are you sure about that? From what I see qbit settings set the folder you provide as root one. So in this case the root is always the `public` folder we provide in the `alternative UI` setting. Also, you can take a look at the build `index.html`. For example `js/app{someHash}.js`, especially show that the root is our `public` folder.

Reply:
@bbogdanov It's a bit complicated.
If url is `/apple.png`, it won't works when deployed with nginx to `example.com/qbit/`, the `/apple.png` will resolved to `example.com/apple.png`. You may already know that.
If url is `apple.png`, it will works, since the url will resolved to `example.com/qbit/apple.png`. It dose works in default release, and it looks nice, so what's the issue?
Somebody(like me) will deploy static files to `cdn.example.com`, but access site in `qb.example.com`(or other domain and path be different to static files). In this case, the url of picture will be wrong.

So, keep it will be better.

Reply:
Just doesn't see a benefit for it when there is no `BASE_URL` provider in the build script. Looks like an overhead at this time. What I am saying here is that we can check that and see if we really need it and maybe remove it or add the possibility to provide a `BASE_URL` when running the build script and have a default one when there is `BASE_URL` provided. 

P.S. Yea, I had to say in the previous comment that I meant to remove the slash as well. `<%= BASE_URL %>/`

Reply:
It's predefined in vue-cli, please check https://cli.vuejs.org/guide/mode-and-env.html#using-env-variables-in-client-side-code